components/cluster/cluster-glue/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2013 Adam Stevko <adam.stevko@gmail.com>
+# Copyright 2016 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= cluster-glue
+COMPONENT_VERSION= 1.0.12
+COMPONENT_FMRI= application/cluster/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION= System/Services
+COMPONENT_HG_REV= 0a7add1d9996
+COMPONENT_SRC= Reusable-Cluster-Components-glue--$(COMPONENT_HG_REV)
+COMPONENT_ARCHIVE= $(COMPONENT_HG_REV).tar.bz2
```

Comment:
Can we have COMPONENT_ARCHIVE = $(COMPONENT_NAME)-$(COMPONENT_HG_REV).tar.bz2
and COMPONENT_ARCHIVE_URL=http://hg.linux-ha.org/glue/archive/$(COMPONENT_HG_REV).tar.bz2 ,
so that archive file had user-friendly name?

And one more style issue: prefer $() to ${}